update.sh

```diff
@@ -24,26 +24,16 @@ then
     exit 69
   fi
 else
-  if type python2 &>/dev/null
+  if type python3 &>/dev/null
   then
-    python_command=python2
-  elif type python &>/dev/null
-  then
-    python_command=python
```

Comment:
since we have the virutal env being crated from upadate.sh which specifies the venv being py3 even the python shebang line from mbrng/ would default to using that itself

Reply:
Alright but then the error message error below should refer to `python3` instead of `python`.

Reply:
fixed.